project_national/fact_sheets_category_4.yml

```diff
@@ -0,0 +1,265 @@
+            multiplier: Size, Water Heater (gal)
+        lifetime: 12
+      - &water_heater_efficiency_electric_heat_pump_80_gal_3_pt_45_uef
+        option: Water Heater Efficiency|Electric Heat Pump, 80 gal, 3.45 UEF
+        apply_logic:
+          - or:
+            - Bedrooms|5
+        costs:
+          - value: 100.0 #TBD
+            multiplier: Size, Water Heater (gal)
+        lifetime: 12
+
+  - upgrade_name: 96% AFUE Natural Gas Furnace
+    options:
+      - &Furnace_fuel_type
+        option: HVAC Heating Type And Fuel|Natural Gas Fuel Furnace
+      - &96%AFUE_furnace
+        option: HVAC Heating Efficiency|Fuel Furnace, 96% AFUE
+        apply_logic:
+          - or:
+            - HVAC Heating Efficiency|Fuel Furnace, 60% AFUE
+            - HVAC Heating Efficiency|Fuel Furnace, 68% AFUE
+            - HVAC Heating Efficiency|Fuel Furnace, 72% AFUE
+            - HVAC Heating Efficiency|Fuel Furnace, 76% AFUE
+            - HVAC Heating Efficiency|Fuel Furnace, 80% AFUE
+            - HVAC Heating Efficiency|Fuel Furnace, 85% AFUE
+            - HVAC Heating Efficiency|Fuel Furnace, 90% AFUE
+            - HVAC Heating Efficiency|Fuel Furnace, 92.5% AFUE
+    package_apply_logic: 
+      - and: 
+        - or:
+          - Heating Fuel|Natural Gas
+          - Water Heater Fuel|Natural Gas
+          - Clothes Dryer|Gas
+          - Clothes Dryer|Gas, Premium
+          - Misc Hot Tub Spa|Natural Gas
+          - Misc Pool Heater|Natural Gas
+          - Misc Gas Fireplace|Gas Fireplace
+          - Misc Gas Grill|Gas Grill
+          - Misc Gas Lighting|Gas Lighting
+        - or:
+          - HVAC Heating Type And Fuel|Fuel Oil Fuel Furnace
+          - HVAC Heating Type And Fuel|Propane Fuel Furnace
+          - HVAC Heating Type And Fuel|Other Fuel Fuel Furnace
+
+  - upgrade_name: 96% AFUE Natural Gas Boiler
+    options:
+      - &Furnace_fuel_type
+        option: HVAC Heating Type And Fuel|Natural Gas Fuel Boiler
+      - &96%AFUE_boiler
+        option: HVAC Heating Efficiency|Fuel Boiler, 96% AFUE
+        apply_logic:
+          - or:
+            - HVAC Heating Efficiency|Fuel Boiler, 72% AFUE
+            - HVAC Heating Efficiency|Fuel Boiler, 76% AFUE
+            - HVAC Heating Efficiency|Fuel Boiler, 80% AFUE
+            - HVAC Heating Efficiency|Fuel Boiler, 82% AFUE
+            - HVAC Heating Efficiency|Fuel Boiler, 85% AFUE
+            - HVAC Heating Efficiency|Fuel Boiler, 90% AFUE
+            - HVAC Heating Efficiency|Fuel Boiler, 95% AFUE, OAT Reset
+    package_apply_logic: 
+      - and: 
+        - or:
+          - Heating Fuel|Natural Gas
+          - Water Heater Fuel|Natural Gas
+          - Clothes Dryer|Gas
+          - Clothes Dryer|Gas, Premium
+          - Misc Hot Tub Spa|Natural Gas
+          - Misc Pool Heater|Natural Gas
+          - Misc Gas Fireplace|Gas Fireplace
+          - Misc Gas Grill|Gas Grill
+          - Misc Gas Lighting|Gas Lighting
+        - or:
```

Comment:
These should be boilers, also add NG boiler

Reply:
Done